src_c/camera.h

```diff
@@ -200,6 +240,21 @@ int mac_copy_gworld_to_surface(pgCameraObject* self, SDL_Surface* surf);
 void flip_image(const void* image, void* flipped_image, int width, int height,
                 short depth, int hflip, int vflip);
 
+#elif defined(PYGAME_WINDOWS_CAMERA)
+/* internal functions specific to WINDOWS */
+WCHAR** windows_list_cameras(int* num_devices);
+int windows_init_device(pgCameraObject* self);
```

Comment:
Something that should be done at some point is change the functions to use the C API prefixes. eg `windows_open_device` should be `pg_windows_open_device`. Don't feel like you have to change that in this PR if you don't want to. Just mentioning it. The other functions/symbols in this module haven't been converted to the C API either yet.

Reply:
I was considering converting to a standard API (different than the user facing interface) in a future addition. The windows backend, an opencv backend I've developed could share a blocking API where the python side takes care of the threading and thread validation things. But I'd like to manage that in the future, along with the prefix, rather than in this PR.